website/content/docs/platform/k8s/vso/examples.mdx

```diff
@@ -11,6 +11,34 @@ The Operator project provides the following examples:
 - Sample use-cases are documented [here](https://github.com/hashicorp/vault-secrets-operator#samples)
 - A Terraform based demo can be found [here](https://github.com/hashicorp/vault-secrets-operator/tree/main/demo)
 
+## Configuring the JWT auth method to avoid API calls to the Kubernetes API Server
+
+Vault Secrets Operator supports using the [JWT auth method](/vault/docs/platform/k8s/vso/api-reference#vaultauthconfigjwt).
+In the default JWT configuration this involves an API call from the Vault server back to the Kubernetes API server to [verify](/vault/docs/auth/jwt#jwt-verification) the JWT.
+It is possible to configure Vault to use [`jwt_validation_pubkeys`](/vault/api-docs/auth/jwt#jwt_validation_pubkeys) fields of the JWT auth configuration
+to work in the Kubernetes environment and avoid the API call back to Kubernetes, improving performance of login requests.
+
+### Configure the JWT Auth method in Vault to use the JWKS keys from Kubernetes:
+
+* Fetch the JWKS key from Kubernetes using `kubectl`
+
+```shell
+demo $ kubectl get --raw "$(kubectl get --raw /.well-known/openid-configuration | jq -r '.jwks_uri' | sed -r 's/.*\.[^/]+(.*)/\1/')" | jq  '.keys | .[0]'
+{
+  "use": "sig",
+  "kty": "RSA",
+  "kid": "o2Tjqi7n9ye3VW6L8....",
+  "alg": "RS256",
+  "n": "2SixHcuzdYrj.....",
+  "e": "AQAB"
+}
+````
+
+* Copy/paste this JSON object into your favourite JWKS->PEM convertor and use this PEM file as the
```

Comment:
Any examples of a specific tool that the user could use? It would be great if we could have a complete pipeline example.

Reply:
Honestly I don't have a great one to link to, there are hundreds of websites that are laden with ads and spam that will do the conversion, and then no real "community accepted tool" that I've been able to find either :-/
I'd feel linking to any of the ones I found probably inappropriate.

Reply:
fixed in ac8dfc66f9